output/schema/import-namespace-graph.json

```diff
@@ -0,0 +1,10954 @@
+[
+  {
+    "namespace": "_global.bulk",
```

Comment:
https://github.com/elastic/elasticsearch-specification/pull/1261#discussion_r784709089 assume the same holds true for the namespace. Should be unique